slides.qmd

```diff
@@ -23,10 +23,73 @@ format:
 
 ## Motivation
 
-*Andreas to add content*
+Learning from others: the good, the bad, and the ugly.
+
+Visualisations are encountered everywhere.
+
+
+## Case study 1: Football tickets ##
+
+![](images/Case study-stadium ticket 1.png){fig-alt="Stadium map"}
+
+- Hovering shows available tickets and prices.
+
+- Select Block A3. Note the positioning of pitch and block.
+
+
+## Case study 1: Football tickets cont'd ##
+
+Select your seat. 
+
+![](images/Case study-stadium ticket 2.png){fig-alt="Stadium map sector"}
+
+- The map is upside down now (pitch at bottom)!
+
+- Seats available are marked red!
+
+## Case study: stadium ticket ##
+
+Messages:
```

Comment:
This content doesn't fit on the slide - can the image at the bottom go (i) into a two column layout, or (ii) onto a new page?